x-pack/test/functional/services/ml/api.ts

```diff
@@ -754,5 +879,59 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       await this.waitForDFAJobTrainingRecordCountToBePositive(dfaConfig.id);
       await this.waitForAnalyticsState(dfaConfig.id, DATA_FRAME_TASK_STATE.STOPPED);
     },
+
+    async asignJobToSpaces(
+      jobId: string,
+      jobType: 'anomaly-detector' | 'data-frame-analytics',
```

Comment:
this could be `JobType` from `../../../../plugins/ml/common/types/saved_objects.ts`

Reply:
Done in 3640b76